tests/miral/resize_window.cpp

```diff
@@ -103,10 +103,10 @@ struct ResizeWindow : mt::TestWindowManagerTools, WithParamInterface<ResizeParam
         Size old_size = window.size();
         Point result = window.top_left();
 
-        if (edge | mir_resize_edge_west)
+        if (edge & mir_resize_edge_west)
```

Comment:
Interestingly, the test results are unchanged because the same code is used to generate the requested change and the expected change.